tck/old-tck/source/src/com/sun/ts/tests/jsf/spec/el/managedbean/common/TestServlet.java

```diff
@@ -29,15 +30,16 @@
 import com.sun.ts.tests.jsf.common.servlets.HttpTCKServlet;
 import com.sun.ts.tests.jsf.common.util.JSFTestUtil;
 
+
 import jakarta.faces.application.Application;
 import jakarta.faces.context.ExternalContext;
 import jakarta.faces.context.FacesContext;
+import jakarta.el.ELManager;
 import jakarta.el.ValueExpression;
 import jakarta.servlet.ServletConfig;
 import jakarta.servlet.ServletException;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import jakarta.servlet.jsp.JspFactory;
 import jakarta.servlet.jsp.JspWriter;
 import jakarta.servlet.jsp.PageContext;
```

Comment:
All jakarta.servlet.jsp imports can be removed since their usage has also been removed.